articles/libraries/machine_learning/concepts/Qccc.Training.md

```diff
@@ -0,0 +1,56 @@
+
+
+# Classifier Training Process and Results
+
+Training of a circuit-centric quantum classifier is a process with many moving parts that require the same (or slightly larger) amount of
+calibration by trial and error as training of traditional classifiers. Here we define the main concepts and ingredients of this training process. 
+
+# Training/testing schedules
+The the context of classifier training a `schedule` describes a subset of data samples in an overall training or testing set. A schedule is usually defined as a
+collection of sample indices.
+
+# Parameter/bias scores
+Given a candidate parameter vector and a classifier bias, their `validation score` is measured relative to a chosen validation schedule S and is expressed by a number of misclassifications
+ counted over all the samples in the schedule S
+
+
+# Hyperparameters
+The model training process is governed by certain pre-set values called `hyperparameters`:
+
+## Learning rate (LR) 
+
+is one of key hyperparameters. It defines how much current stochastic gradient estimate impacts the parameter update. The size of parameter update delta is 
+proportional to LR. Smaller LR values lead to slower parameter evolution and slower convergence, but excessively large values of LR may break the convergence 
```

Comment:
Are the abbreviations for each hyperparameter needed? I don't think we use them in the APIs, but rather use the full name for each hyperparameter, such that it may be more clear to do so here as well.

Reply:
I use abbreviations for hyperparameters to later shorthand them in narration. If you don't like this style I can easily expand the abbreviations back.

Reply:
My concern is in making sure that the docs are as accessible as possible, especially given that readers may enter at pretty arbitrary points. If you do want to use abbreviations, I'd suggest using appropriate metadata like [`<abbrev>`](https://www.w3schools.com/TAGS/tag_abbr.asp) to help remind readers of definitions.

Reply:
I have undone the abbreviations